scripts/test/unit.sh

```diff
@@ -0,0 +1,5 @@
+pytest \
+    -vv \
+    --durations=0 \
+    -m "not (integration or perf)" \
+    --ignore=tests/test_example_dags.py
```

Comment:
```suggestion
    --durations=0
```
We do not have integration or perf ATM